nat-lab/tests/test_events.py

```diff
@@ -398,10 +398,11 @@ async def test_event_content_exit_through_peer(
 
         await testing.wait_long(client_beta.get_router().create_exit_node_route())
 
-        await testing.wait_long(client_alpha.connect_to_exit_node(beta.public_key))
-
         await testing.wait_long(
-            client_alpha.wait_for_event_peer(beta.public_key, [State.Connected])
+            asyncio.gather(
+                client_alpha.connect_to_exit_node(beta.public_key),
+                client_alpha.wait_for_event_peer(beta.public_key, [State.Connected]),
+            )
```

Comment:
Hmm, maybe it would be more intuitive to first connect to beta node and then make it an exit node?

Reply:
you mean `client_alpha.connect_to_exit_node(beta.public_key)` before `client_beta.get_router().create_exit_node_route()`?

Reply:
I think he meant something like this:

```
async with run_async_context(client_alpha.wait_for_event_peer(beta.public_key, [State.Connected])) as future:
    await testing.wait_long(client_alpha.connect_to_exit_node(beta.public_key))
    await testing.wait_long(future)
```

But putting them to `asyncio.gather` is already enough